app/controllers/chargeback_controller.rb

```diff
@@ -725,6 +725,21 @@ def get_tags_all(category)
     classification.entries.each { |e| @edit[:cb_assign][:tags][e.id.to_s] = e.description } if classification
   end
 
+  def get_sub_tenants(tenant, parent)
```

Comment:
@aljesusg and basically can you how to use https://github.com/stefankroes/ancestry here ?
because we are using it 
Tenant.first.subtree works https://github.com/stefankroes/ancestry#navigating-your-tree for example